src/main/java/com/github/theholywaffle/teamspeak3/TS3Api.java

```diff
@@ -2695,6 +2695,24 @@ public boolean moveClient(int clientId, int channelId) {
 	}
 
 	/**
+	 * Moves multiple clients into the specified channel.
+	 *
+	 * @param clientIds
+	 * 		the list of client ID's to move
```

Comment:
The plural of ID is IDs without apostrophe. This line could also profit from rephrasing.